terraform/example.tf

```diff
@@ -0,0 +1,63 @@
+terraform {
+  required_version = "0.11.8"
+}
+
+provider "aws" {
+  version = "1.33.0"
+  region  = "eu-west-1"
+}
+
+# Use the default VPC and subnets
+data "aws_vpc" "main" {
+  default = true
+}
+
+data "aws_subnet_ids" "main" {
+  vpc_id = "${data.aws_vpc.main.id}"
+}
+
+# Use the latest Amazon Linux 2 AMI
+data "aws_ami" "linux2" {
+  owners      = ["amazon"]
+  most_recent = true
+
+  filter {
+    name   = "virtualization-type"
+    values = ["hvm"]
+  }
+
+  filter {
+    name   = "architecture"
+    values = ["x86_64"]
+  }
+
+  filter {
+    name   = "root-device-type"
+    values = ["ebs"]
+  }
+
+  filter {
+    name   = "name"
+    values = ["amzn2-ami*gp2"]
+  }
+}
+
+module "ssm-example" {
+  source = "modules/ssm-example"
+
+  name_prefix = "ssm-sh-example"
+  vpc_id      = "${data.aws_vpc.main.id}"
+  subnet_ids  = ["${data.aws_subnet_ids.main.ids}"]
+
+  instance_ami   = "${data.aws_ami.linux2.id}"
+  instance_count = "1"
+  instance_type  = "t3.micro"
+
+  output_bucket    = "<bucket-name>"
```

Comment:
Should the example create a bucket and log group so you can launch this example directly?

Reply:
I think I originally decided against generating the bucket for fear of conflicting bucket names, but I suppose we could do it and just prefix the bucket with the account ID?

Reply:
Yepp, I think it would be nice to have everything in the example to test this. If we use account ID prefix it should be fine!